packages-new/components/range/src/components/osds-range/public-api.ts

```diff
@@ -1,2 +1,3 @@
 export type { OdsRangeAttribute } from './interfaces/attributes'
+export type { OdsRangeEvent } from './interfaces/events'
```

Comment:
and the rest `OdsRangeValueChangeEventDetail`, `interface OdsRangeValueChangeEvent` so that people can use those type when managing range event on their side

Reply:
fixed